src/components/Modals/EditExpenseModal.js

```diff
@@ -0,0 +1,90 @@
+import DatePicker from 'react-datepicker';
+import { Modal } from './components/Modal';
+import { useState, useRef } from 'react';
+import { convertCentsToEuros, convertEurosToCents } from '../../utils/helpers';
+
+const EditExpenseModal = ({ setIsOpen, expense, editExpense }) => {
```

Comment:
the structure of this whole file looks very much like a class.

Reply:
I'm not sure if I follow. Could you elaborate?

Reply:
My seniors share the same feeling as yours @motine ... When I was working on my Modals I also had the same feeling ... My personal opinion is that React functional components are yes  a bit messier but still easier to use compared to React class components. Moreover, for a beginner the learning curve is much easier to overcome.